### ISSUE TITLE
Saila-Lorda and sigmoidal Beverton-Holt as future additional SR in Synthesis

### ISSUE THREAD

**Issue by akatan999** [nmfs-stock-synthesis/stock-synthesis#253]
Recent paper from Perala eta al., showed that depensation is more common than previously assumed in fish. It would be great to allow in the future for depensatory SR curves in Synthesis as for example:

Saila-Lorda = αSγ exp(−βS)

Sigmoidal Beverton-Holt = αSγ/1+ βSγ

where S=SSB and γ is the compensatory parameter

**Comment by k-doering-NOAA** [comment]
Thanks @akatan999! Could you provide the doi for the recent paper?

**Comment by Rick-Methot-NOAA** [comment]
That's not the correct DOI.
use:
Detection of Allee effects in marine fishes: Analytical biases generated by data availability and model selection
August 2017Proceedings of the Royal Society B: Biological Sciences 284(1861):20171284
DOI:10.1098/rspb.2017.1284
Tommi Perälä
Anna Kuparinen

**Comment by akatan999** [comment]
True, it is the most recent paper with the inclusion of those two SR into SAM model. I will soon upload the second paper from Perala about Allee effect in cod, which is in press on Biology Letters,
